scipy/linalg/_decomp.py

```diff
@@ -1219,6 +1220,9 @@ def eigh_tridiagonal(d, e, eigvals_only=False, select='a', select_range=None,
         The diagonal elements of the array.
     e : ndarray, shape (ndim-1,)
         The off-diagonal elements of the array.
+    eigvals_only : bool, optional
+        Compute only the eigenvalues and no eigenvectors.
+        (Default: calculate also eigenvectors)
```

Comment:
Default is `False` not `calculate also eigenvectors`

Reply:
This was copied from `eig_banded`, but sure.